packages/patternfly-4/react-core/src/components/Brand/Brand.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Additional classes added to the Brand. */
+  className: PropTypes.string,
+  /** Attribute that specifies the URL of the image for the Brand. */
+  src: PropTypes.string,
+  /** Attribute that specifies the alt text of the image for the Brand. */
+  alt: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  src: ''
+};
+
+const Brand = ({ className, src, alt, ...props }) => (
+  <img {...props} className={css('pf-c-brand', className)} src={src} alt={alt} />
```

Comment:
can you add a comment to say why we are using the 'pf-c-brand' string here, incase someone else is questioning it.